### DISCUSSION TITLE
Very minor, but the calculation for "last week" seems off

### DISCUSSION THREAD

**Discussion by kalafut** [community/community#46534] [Pull Requests]
### Select Topic Area

Bug

### Body

Today is Monday, **2023-02-06**. I'm reviewing a commit that was merged "last week" (per the link text), but the tooltip shows the actual merge date as **2023-01-27**. I know dates and calendar math have a lot of variability, but I'm struggling to see how this makes sense since it is both not in the last calendar week nor within 7 days.

Minor to be sure, but it did trip me up when I thought "surely we got in before last week" (and we had...)

<img width="335" alt="image" src="https://user-images.githubusercontent.com/211880/217045301-dc50acca-ceba-4672-9288-c08a65c0bc89.png">

**Comment by Doloment** [comment]
The date calculation in general. Same problem but it says last year for some reason
![image](https://user-images.githubusercontent.com/22934348/217125040-12bed2d7-6a19-49a6-be86-ad8efff74145.png)
